tests/test_pandas_coverage.py

```diff
@@ -595,7 +595,12 @@ def test_df_to_series(global_var):
         "_prev",
         "_history",
         "_saved_export",
```

Comment:
Can we share this list of variables to check so that we don't have to update all of them across the different test functions?

Reply:
Yes, I'll add that!

Reply:
Set a variable in `conftest` for the metadata list, so we just have to edit that list.